core/vm/gas.go

```diff
@@ -30,10 +30,6 @@ const (
 	GasMidStep     uint64 = 8
 	GasSlowStep    uint64 = 10
 	GasExtStep     uint64 = 20
```

Comment:
Hmm, why are these even defined in this file? @holiman any ideas? Shouldn't we move everything to `params`?

Reply:
Hh, yeah, no idea why those are there. Looks odd